04/js/app.js

```diff
@@ -0,0 +1,43 @@
+// 1. Funkcja z 20 eleemtami
+
+function createArr(min, max) {
+  const newArr = [];
+
+  for (let i = min; i <= min + 19; i++) {
+    let maxNum = Math.round(Math.random() * max);
+
+    newArr.push(maxNum);
```

Comment:
Zwróć uwagę, żę w zadaniu nr 3 miałaś kod, który losuje liczby z konkretnego przedziału tj. `Math.round(Math.random() * (max - min) + min);`. Jeśli nie jest on dla Ciebie zrozumiały to podstaw sobie konkretne liczby pod zmienne tj. `max=4`, `min=0` oraz `max=3` i `min=3` i zobacz co tam się może pojawić.